be/src/exec/pipeline/limit_operator.h

```diff
@@ -48,23 +50,26 @@ class LimitOperator final : public Operator {
     bool _is_finished = false;
     std::atomic<int64_t>& _limit;
     ChunkPtr _cur_chunk = nullptr;
+    bool _limit_chunk_in_place;
```

Comment:
I couldn't grasp the meaning of it at first glance; adding some comments here would be helpful.